fs/vfs/fs_stat.c

```diff
@@ -58,7 +58,8 @@
  ****************************************************************************/
 
 static inline int statroot(FAR struct stat *buf);
-int stat_recursive(FAR const char *path, FAR struct stat *buf, int resolve);
+static int
+stat_recursive(FAR const char *path, FAR struct stat *buf, int resolve);
```

Comment:
while nxstyle will pass on this I believe the preferred way to handle this is to keep the type on the same line as the function name and then drop the last argument onto the next line aligned with the first argument.

Reply:
Done.